src/subword.rs

```diff
@@ -297,33 +297,55 @@ impl<'a> Iterator for NGrams<'a> {
 }
 
 /// Extension trait for computing subword indices.
-///
```

Comment:
This information is indeed outdated. However, it would be nice to have more than the current blurb. Maybe something about the requirements on implementors (e.g. collisions are permitted).